rules/0390-fortigate_rules.xml

```diff
@@ -105,9 +105,6 @@ ID: 81600 - 80799
         <if_matched_sid>81610</if_matched_sid>
         <same_source_ip />
         <description>Fortigate: Multiple default tunneling setting events from same source.</description>
```

Comment:
Not sure about this one: T1562.004 Impair Defenses: Disable or Modify System Firewall